mysql-test/suite/innodb/t/doublewrite.test

```diff
@@ -206,16 +110,18 @@ unlock tables;
 begin;
 insert into t1 values (6, repeat('%', 400));
 
---source ../include/no_checkpoint_start.inc
+set global innodb_log_checkpoint_now = 1;
 
 --echo # Make the 2nd page dirty for table t1
 set global innodb_saved_page_number_debug = 1;
 set global innodb_fil_make_page_dirty_debug = @space_id;
 
---echo # Ensure that dirty pages of table t1 are flushed.
-set global innodb_buf_flush_list_now = 1;
+SET GLOBAL innodb_max_dirty_pages_pct_lwm=0.0;
+SET GLOBAL innodb_max_dirty_pages_pct=0.0;
 
---source include/no_checkpoint_end.inc
+sleep 1;
+let $shutdown_timeout=0;
+--source include/shutdown_mysqld.inc
```

Comment:
It is unclear to me why the write of page 1 of `t1` would be observed after the server is killed. Why would this `sleep 1` not typically wait for a checkpoint to be executed? In that case, crash recovery should parse an empty undo log.